pkg/runtime/framework/plugins/jobset/jobset.go

```diff
@@ -125,7 +126,12 @@ func (j *JobSet) ReconcilerBuilders() []runtime.ReconcilerBuilder {
 	}
 	return []runtime.ReconcilerBuilder{
 		func(b *builder.Builder, cl client.Client, cache cache.Cache) *builder.Builder {
-			return b.Owns(&jobsetv1alpha2.JobSet{})
+			return b.Watches(
```

Comment:
Why are we changing we it ?

Reply:
The typed manager requires `Watches` since it does not provide `Owns`.